class.ext_update.php

```diff
@@ -61,4 +78,82 @@ public function main()
 		return '<pre>' . $output . '</pre>' . PHP_EOL .
 			sprintf( "Setup process lasted %1\$f sec</br>\n", (microtime( true ) - $exectimeStart) );
 	}
+
+	/**
+	 * Check the environment.
+	 *
+	 * TYPO3 9.5 and MySql 5.6 and below may make problems. This was introduced
+	 * with #80398.
+	 *
+	 * @return bool
+	 */
+	protected function checkEnvironment(): bool
+	{
+		if ( version_compare( TYPO3_version, '9.5', '<' ) )
+		{
+			// Wrong TYPO3 version.
+			return true;
+		}
+
+		/** @var ObjectManager $objectManager */
+		$objectManager = GeneralUtility::makeInstance( ObjectManager::class );
+
+		/** @var ConnectionPool $connectionPool */
+		$connectionPool = $objectManager->get( ConnectionPool::class );
+		// A join with tables from different databases will throw an exception.
+		// Hence, it is extreme likely that aimeos will be installed on the same
+		// database where the fe_user is located.
+		$connection = $connectionPool
+			->getQueryBuilderForTable( 'fe_user' )
+			->getConnection();
+
+		// Test the server type
+		// @todo test MariaDB
+		// According to the releases, the next version here would be 10.0
+		if ( strpos( $connection->getServerVersion(), 'MySQL' ) === 0 &&
```

Comment:
Here, the check for MariaDB will require different values

Reply:
The string 'MySql' comes hardcoded from the database connection class, handling the platforms 'mysql', 'pdo_mysql' and 'drizzle_pdo_mysql'.
Fallback would be the raw name from the platform driver. The MariaDB driver extends the MySql platform, so my guess was that both would return the same getName(), since the MariaDB does not implement said method.

So, what would the name of the MariaDB be?

Reply:
@vmans Can you help with name and version?